src/libaktualizr/uptane/imagesrepository.cc

```diff
@@ -235,6 +241,11 @@ bool ImagesRepository::updateMeta(INvStorage& storage, Fetcher& fetcher) {
       local_version = -1;
     }
 
+    // I am not sure that #6 of https://uptane.github.io/uptane-standard/uptane-standard.html#check_snapshot is
+    // performed
```

Comment:
What's your concern? That we don't match the Targets metadata filenames?

Reply:
Yes, I mean exactly what #6 says

"Check that each Targets metadata filename listed in the previous Snapshot metadata file is also listed in this"

Reply:
This might be another thing worth making a ticket for and looking into more. I'm not sure how critical this really is, though, and I'm also not sure that the backend doesn't make some optimizations that would prevent that from working. In fact, I'm not sure this is actually sensible; what happens if a delegation gets (legitimately) removed?

Reply:
FYI: https://github.com/uptane/uptane-standard/issues/149. Let's wait for resolution on that before making changes on our side.

Reply:
OK